src/mono/System.Private.CoreLib/src/System/Reflection/Emit/ILGenerator.Mono.cs

```diff
@@ -947,20 +947,6 @@ public virtual void MarkLabel(Label loc)
                 cur_stack = labels[loc.m_label].maxStack;
         }
 
-        public virtual void MarkSequencePoint(ISymbolDocumentWriter document, int startLine,
-                               int startColumn, int endLine, int endColumn)
-        {
-            if (currentSequence == null || currentSequence.Document != document)
-            {
-                if (sequencePointLists == null)
-                    sequencePointLists = new List<SequencePointList>();
-                currentSequence = new SequencePointList(document);
```

Comment:
Is the `SequencePointList` class completely unused now? Could oyu remove it, too?

(Also the commented out `GenerateDebugInfo`, below)

Reply:
Removed